# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/RtpRtcpConfig.h
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/libwebrtcglue/RtpRtcpConfig.h
--- a/dom/media/webrtc/libwebrtcglue/RtpRtcpConfig.h	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/libwebrtcglue/RtpRtcpConfig.h	Fri Oct 23 14:02:54 2020 -0400
@@ -4,7 +4,7 @@
 
 #ifndef __RTPRTCP_CONFIG_H__
 #define __RTPRTCP_CONFIG_H__
-#include "webrtc/common_types.h"
+#include "api/rtp_headers.h"
 
 namespace mozilla {
 class RtpRtcpConfig {